packages/webpack-loader/index.js

```diff
@@ -0,0 +1,18 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
You can remove this and the css entry if you add this below `resolve` in the `test-utils` file:

```
resolveLoader: {
  alias: {
    '@compiled/webpack-loader/css-loader': join(__dirname, '..', 'css-loader.tsx'),
    '@compiled/webpack-loader': join(__dirname, '..', 'index.tsx'),
  }
},
```

Reply:
Did you have any luck trying this? I did it locally and couldn't get it working - I always assumed this was for aliasing during bundling not bootstrapping 🤔.

Reply:
Yeah it worked for me when I tried it out. Putting it in `resolve` didn't work, but `resolveLoader` was fine